### DISCUSSION TITLE
No Royal option for ini files?

### DISCUSSION THREAD

**Discussion by jaakezzz** [fairy-stockfish/Fairy-Stockfish#527] [Ideas]
This is like the one option you actually need to create variants, and seems so simple.

I'm dying to get a functioning Stockfish engine for this brilliantly designed variant:
https://www.chess.com/join/dragonanga

The chess.com customs team failed to figure out the strategies of the variant, making it far more intriguing since I have discovered multiple effective strategies. This is why I'm eager to evaluate the game with an engine. Anyone that can help would be greatly appreciated. :)

**Comment by ianfab** [comment]
For custom royal piece movement, see https://github.com/ianfab/Fairy-Stockfish/blob/master/src/variants.ini#L93-L98.

**Reply by jaakezzz** [reply]
oh I get it. I thought that was just a custom piece example.

So what is the notation to make the King a Ferz? I'd want to remove Wazir movement from the King I suppose?

Then I could make a non-royal King by combining the Ferz and Wazir?  
`customPiece1 = F:W`
